test/Interop/Cxx/stdlib/avoid-import-cxx-math.swift

```diff
@@ -6,18 +6,13 @@ import CxxStdlib
 
 func test() {
     let x: Float = 1.0
-    let y: Double = 2.0
+    let y: Float = 2.0
 
-    // Note: we dispatch `pow(Float,Double)`
-    // to ensure we don't pick up the
-    // C++ stdlib `pow` function template.
-    // The `pow` function is still reexported
-    // from Darwin via CxxStdlib, so there are
-    // matching overloads that can be found still.
-    // Note: the error is different on Glibc instead
-    // of Darwin, so do not check the exact error.
-    let _ =  CxxStdlib.pow(x, y) // expected-error {{}}
+    let _ =  pow(x, y)
 
-    let _ = CxxStdlib.abs(x) // expected-error {{module 'CxxStdlib' has no member named 'abs'}}
```

Comment:
I was getting errors like this:
```
Darwin.pow:1:13: note: diagnostic produced elsewhere: candidate expects value of type 'Double' for parameter #1 (got 'Float')
public func pow(_: Double, _: Double) -> Double
```
So I figured we could just test the positive case.

Reply:
Also testing the positive case seems more direct in terms of making sure the thing we want to work works. It also is a bit less fragile and more portable.